pkg/volume/emptydir/quota.go

```diff
@@ -84,6 +94,11 @@ func (cr *realQuotaCommandRunner) RunApplyQuotaCommand(fsDevice string, quota re
 	return "", stderr.String(), err
 }
 
+func (cr *realQuotaCommandRunner) RunMountOptionsCommand(dir string) (string, string, error) {
+	outBytes, err := exec.Command("mount").Output()
```

Comment:
Oh my mistake, I read output instead of input. Yes you're both right, dir is unused and unnecessary. I'll kill it.